library/ui/src/main/java/com/google/android/exoplayer2/ui/AspectRatioFrameLayout.java

```diff
@@ -51,6 +51,10 @@
    * The specified aspect ratio is ignored.
    */
   public static final int RESIZE_MODE_FILL = 3;
+  /**
+   * The height or width is increased or decreased to crop and to obtain the desired aspect ratio.
```

Comment:
Should this be "Either the width or height is increased to obtain the desired aspect ratio" :)?